geth/account/interfaces.go

```diff
@@ -0,0 +1,45 @@
+package account
+
+import (
+	"crypto/ecdsa"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/extkeys"
+)
+
+type accountNode interface {
+	AccountKeyStore() (accountKeyStorer, error)
+	AccountManager() (gethAccountManager, error)
+	WhisperService() (whisperService, error)
+}
+
+type gethAccountManager interface {
```

Comment:
Oh, I see, package private interfaces? This is something I seldom have seen before too. Also missing documentation here.

Reply:
Agree. I'll add it.

Reply:
@themue As you could see, i've go second way.